riscv/decode.h

```diff
@@ -250,6 +273,7 @@ class regfile_t
 #define require_either_extension(A,B) require(p->extension_enabled(A) || p->extension_enabled(B));
 #define require_impl(s) require(p->supports_impl(s))
 #define require_fp          require(STATE.sstatus->enabled(SSTATUS_FS))
+#define require_fp_or_zfinx require(p->extension_enabled(EXT_ZFINX) || STATE.sstatus->enabled(SSTATUS_FS))
```

Comment:
This should not be necessary since 66c4853bdc5b22bf4c4b364218c713e3f1e487f3; if `mstatus.FS` does not exist, `require_fp` will do nothing.